modular/executor/execute_replicate.go

```diff
@@ -302,7 +313,9 @@ func (e *ExecuteModular) doneReplicatePiece(ctx context.Context, rTask coretask.
 	receive.SetSignature(taskSignature)
 	doneReplicateTime := time.Now()
 	if err = retry.Do(func() error {
```

Comment:
Why is there no callback parameter of retry.OnRetry?

Reply:
added